calico-enterprise/operations/logstorage/create-storage.mdx

```diff
@@ -124,7 +124,7 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: tigera-elasticsearch
-provisioner: kubernetes.io/azure-disk
```

Comment:
@ctauchen I think other `kubernetes.io`-prefixed provisioners are deprecated too, not just `kubernetes.io/azure-disk`? there are more such occurences, such as `kubernetes.io/aws-ebs` and `kubernetes.io/gce-pd`

Reply:
see e.g. [this document](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi-migration-faq.html) from AWS

Reply:
Thanks, I've updated `aws-ebs` and `gce-pd`. I can't find anything else, though I am concerned that these external provisioners may require additional steps. Do they need to be separately installed? @moredatapls

Reply:
@ctauchen I am not sure about AWS and GCP but on AKS the providers do have to be activated while provisioning a cluster, see e.g. here: https://learn.microsoft.com/en-us/azure/aks/csi-storage-drivers#enable-csi-storage-drivers-on-an-existing-cluster